packages/core/core/secrets-to-env.js

```diff
@@ -0,0 +1,61 @@
+const getSecretValue = async () => {
+    console.log('Fetching secrets...');
+
+    const httpPort = process.env.PARAMETERS_SECRETS_EXTENSION_HTTP_PORT || 2773;
```

Comment:
Should we really default here? If this is a env var used in more places, we should throw an error if it's not provided.